src/abstractdataframe/io.jl

```diff
@@ -229,7 +229,7 @@ importall DataStreams
 # DataFrames DataStreams implementation
 function Data.schema(df::DataFrame, ::Type{Data.Column})
     return Data.Schema(map(string, names(df)),
-            DataType[typeof(A) for A in df.columns], size(df, 1))
+           DataType[typeof(A) for A in df.columns], size(df, 1))
```

Comment:
No, this should align with `map` since it's the continuation of the arguments list.